src/wwwroot/app/views/reports/users-report/users.report.controller.js

```diff
@@ -158,15 +121,24 @@ export default function UsersReportController(
 
    function filterArrayByProperty(report, prop, stageId) {
       let rep = find(report, {userId: prop});
-      if (rep) {
-         if (rep.stagesData[stageId]) {
-            return rep.stagesData[stageId];
-         } else {
-            return 0;
-         }
-      } else {
-         return 0;
-      }
+      return rep && rep.stagesData[stageId] ? rep.stagesData[stageId] : 0;
+   }
+
+   function isEqualLocations(user) {
+      return vm.usersReportParametrs.locationIds.length ?
+         vm.usersReportParametrs.locationIds.includes(user.cityId) : true;
```

Comment:
if ternar expression is not one line, it is better to make three lines:
condition ?
then_expression : 
else_expression